metabrainz/templates/index/contact.html

```diff
@@ -74,6 +74,7 @@ <h1 class="page-title">Contact</h1>
   <p>
     The MetaBrainz/MusicBrainz team, as well as MusicBrainz users, hang out in the
     <strong><a href="irc://irc.freenode.net/metabrainz">#metabrainz</a></strong>
+    (<a href="https://webchat.freenode.net/?channels=metabrainz">webchat</a>)
     channel on IRC chat at irc.freenode.net. Please feel free to browse the
```

Comment:
This feels confusing to me. I think it would be better to be a little bit more explicit:

If you don't have an IRC client install, you might try this [webchat client].